kiwi/iso.py

```diff
@@ -99,6 +99,11 @@ def create_hybrid(self, offset, mbrid, isofile, efi_mode=False):
         :param string mbrid: boot record id
         :param string isofile: path to the ISO file
         """
+        ignore_errors = [
+            # we ignore this error message, for details see:
+            # http://www.syslinux.org/archives/2015-March/023306.html
+            'Warning: more than 1024 cylinders'
+        ]
```

Comment:
I think there is no good solution for this, however I am wondering if it is safe using an exact match on the warning message, not sure how many different versions of isohybrid are we using in supported distros.

Reply:
I fully agree and also don't like it. If the match fails the warning is still treated as a fatal error which is what I'd like to achieve. I have checked the isohybrid code and this particular warning hasn't changed since ~2years. I don't expect people to massage the error message, I rather expect it to go away which would not impact the code in here.

If we find out that a more fine grained or generic match is needed we could do this in a subsequent patch.

After all I would delete such a code rather yesterday than tomorrow but I don't see another way to distinguish between warnings I consider really errors and other warnings we should not care about.

Currently my load is high but if time permits the real fix should go to syslinux upstream and a correct error handling based on the exit code of isohybrid. Assuming a patch would be accepted we still had to deal with the problem for older versions. Thus this patch here seems kind of inevitable

Thougths ?

Reply:
> Currently my load is high but if time permits the real fix should go to syslinux upstream and a correct error handling based on the exit code of isohybrid. Assuming a patch would be accepted we still had to deal with the problem for older versions. Thus this patch here seems kind of inevitable

fully agree, the _real fix_ does not prevent us of having to code some ugly stuff. Thus let's start with it, then we'll see if we have some spare cycles to provide some feedback/patch to syslinux.